### PR TITLE
Widescreen chat

### PR DESCRIPTION
**Pull request by cnkeats**
The chat window now moves over when widescreen is enabled

### CODE REVIEW

File: Online/Menus/CSS/HandleInputsOnCSS.asm [hunk 1]
```diff
@@ -665,7 +665,15 @@ lwz r3, 0x18(r3) # pointer to our custom bg jobj
...
-lfs f1, TPO_CHAT_WINDOW_X(REG_TEXT_PROPERTIES)
+# Move to the left if widescreen is enabled
+lfs f1, TPO_CHAT_WINDOW_X(REG_TEXT_PROPERTIES) # X POS
+lbz r4, OFST_R13_ISWIDESCREEN(r13)
+cmpwi r4, 0
+beq END_SET_CHAT_WINDOW_POS_X
+lfs f1, TPO_CHAT_WINDOW_X_WIDESCREEN(REG_TEXT_PROPERTIES) # X POS Widescreen
+
+END_SET_CHAT_WINDOW_POS_X:
+lfs f3, TPO_CHAT_LABEL_Y(REG_TEXT_PROPERTIES) # Y POS
```
Comment (rapito) [inline Online/Menus/CSS/HandleInputsOnCSS.asm]: @cnkeats why are you setting f3 to CHAT_LABEL_Y here for?

**Comment by cnkeats** [comment]
image

**Comment by rapito** [comment]
@cnkeats also, the window is too far to the left, it needs to be at a certain distance from the screen bounds so when you move the camera with c-stick it is inside the safe bounds (it does not get hidden) https://github.com/project-slippi/slippi-ssbm-asm/pull/45

This is a safe distance: 
image

File: Online/Menus/CSS/HandleInputsOnCSS.asm (hunk 1 above)
Comment (cnkeats) [reply Online/Menus/CSS/HandleInputsOnCSS.asm]: This was a rogue command and was removed - good catch.

**Comment by cnkeats** [comment]
Moved the chatbox to be inside the safe bounds
image

**Pull request merged**